cypress/e2e/files.cy.ts

```diff
@@ -0,0 +1,75 @@
+// This only works when running against the real backend.
+// Must set the following env vars with real username/pw from local environment:
+
+// export CYPRESS_EMAIL=
+// export CYPRESS_PASSWORD=
+
+Cypress.session.clearAllSavedSessions();
+
+beforeEach(() => {
+  cy.login(Cypress.env('EMAIL'), Cypress.env('PASSWORD'));
+  cy.visit('/');
+});
+
+it(
+  'should allow uploading and deleting client files',
+  {
+    viewportHeight: 1000, // extra long height so its easier to see what's going on
+    viewportWidth: 1024,
+  },
+  () => {
+    // Set up new client
+    cy.createClient('Cy First', 'Cy Last');
+
+    // Got to files tab and create new file
+    cy.get('#side-nav-files').click();
+    cy.get('#add-client-file').click();
```

Comment:
for consistency, can we use test IDs instead of html IDs
```suggestion
    cy.testId('addClientFileButton').click();
```